### PR TITLE
Use compareBeats instead of reimpelement #307

### PR DESCRIPTION
**Pull request by valentinbujdoso**

### CODE REVIEW

File: src/main/java/org/rulez/demokracia/pdengine/Normalizable.java [hunk 1]
```diff
@@ -2,38 +2,30 @@
...
 public interface Normalizable extends ContainingBeats {
 
 	long serialVersionUID = 1L;
 
 	default void normalize() {
-		for (String key : getKeyCollection()) {
+		Collection<String> keys = getKeyCollection();
+		
+		for (String key : keys) {
 			setElement(key, key, new Pair(0, 0));
```
Comment (szirbucz) [inline src/main/java/org/rulez/demokracia/pdengine/Normalizable.java]: It should be moved into the next for.

Comment (valentinbujdoso) [reply src/main/java/org/rulez/demokracia/pdengine/Normalizable.java]: You were faster than me. :)

**Review by magwas — APPROVED** [review]
I love commits which result in less lines of code.

**Comment by magwas** [comment]
testcasecount=1

**Pull request merged**